### DISCUSSION TITLE
starting a replay can be obstructed by user input

### DISCUSSION THREAD

**Discussion by XeZey** [ppy/osu#28939] [Q&A]
### Type

Game behaviour

### Bug description

in the video im spamming enter. When you press "watch replay", the screen that is being shown before entering the replay is interactable. you can also type and press keys or exit the screen during this. although these actions wont prevent the replay from starting.

### Screenshots or videos

https://github.com/user-attachments/assets/7846e537-ff18-4ab0-b8b3-49948cd6fbef

### Version

2024.718.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16286097/compressed-logs.zip)

**Comment by frenzibyte** [comment]
Don't spam enter when starting replay? I'm not sure there should be an issue thread for this.

**Reply by XeZey** [reply]
It seems like there 1 - shouldnt be possible input during this transition, as there is no single good reason for this being possible, as well as typing also throwing unhandled errors, and/or 2 - this transition shouldnt exist in the first place. But I am absolutely no developer so I can just be mistaken.

**Reply by frenzibyte** [reply]
It shouldn't be possible, yes, but it's not worth becoming an "issue" that needs to be "fixed" alongside all of the actual real issues that break the experience right now, that's why I made this a discussion thread. Typing throwing unhandled error is tracked in https://github.com/ppy/osu/issues/21120.
